src/main/java/com/webdev/cosmo/cosmobackend/posts/mapper/PostModelMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.webdev.cosmo.cosmobackend.posts.mapper;
+
+import com.webdev.cosmo.cosmobackend.posts.dto.PostModel;
+import com.webdev.cosmo.cosmobackend.posts.model.Post;
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+@Mapper
+public interface PostModelMapper {
+    PostModel simpleMap(Post post);
+    Post modelToPost(PostModel postModel);
```

Comment:
please, as other comments said, change names to map, where You can! simpleMap? This name is not good. Also modelToPost? rename it